css/selectors/-webkit-scrollbar-button.json

```diff
@@ -36,7 +36,8 @@
               "version_added": "4"
             },
             "safari_ios": {
-              "version_added": "3"
+              "version_added": "3",
+              "version_removed": "14"
```

Comment:
I tried the example on https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar in BrowserStack. It looks like this actually stopped working in version 13. Unless I'm missing something, it looks like this version number needs to be updated throughout the PR (with the exception noted at the end).

Reply:
Yes on BrowserStack it also happens on version 13.4 but I suspect this also affects any iOS 13 version.
Funny nobody mentioned this anywhere yet for iOS 13.
I will update the PR to correct this 😉